cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -24,11 +24,11 @@ The HTTP RESTful API is divided into these sections:
 - :ref:`metadata tags <http-restful-api-metadata-tags>`
 - :ref:`searching properties <http-restful-api-metadata-searching>`
 - :ref:`viewing lineage <http-restful-api-metadata-lineage>`
+- :ref:`metadata for a run of a program <http-restful-api-metadata-run>`
 - :ref:`update notifications <http-restful-api-metadata-notifications>`
 
-Metadata keys, values, and tags must conform to the CDAP :ref:`supported characters
-<supported-characters>`, and are limited to 50 characters in length. Note that tags cannot
-include hyphens or underscores (``- _``) while property keys and values can. The entire
+Metadata keys, values, and tags must conform to the CDAP :ref:`alphanumeric extra extended
+character set <supported-characters>`, and are limited to 50 characters in length. The entire
 metadata object associated with a single entity is limited to 10K bytes in size.
```

Comment:
I don't think we enforced this restriction, because it would require an extra scan. We should remove this.